source/assets/stylesheets/components/_article.scss

```diff
@@ -20,46 +20,49 @@ $_list-indent: $small-spacing * 1.5;
     padding-bottom: $small-spacing;
   }
 
+  p + ol,
+  p + ul {
+    margin-bottom: $small-spacing;
+  }
+
+  li {
+    display: grid;
+    grid-template-columns: $_list-indent auto;
+
+    > * {
```

Comment:
this feels scary, but i'm guessing your environment and lists are controlled enough that you feel comfortable with this?

Reply:
Since column 1 is reserved for the number/symbol, seems reasonable to say "if I'm a list-item, put all of my direct children in the "content" column, ie column 2.